ondemand-log.graphqls

```diff
@@ -0,0 +1,45 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+type Containers {
+    containers: [String!]!
+}
+
+input StreamingLogQueryCondition {
+    namespace: String!
+    serviceId: ID
+    serviceInstanceId: ID
+    container: String!
+    # The time range of log happened
+    # [Required] duration is required in most query, only exception is when use relatedTrace.
+    queryDuration: Duration
+    paging: Pagination!
```

Comment:
2 questions

1. I think the duration is always required as on-demand should not have `relatedTrace` in this case?
2. how does `paging` work with streaming log?